.codacy.yml

```diff
@@ -1,2 +1,3 @@
 ---
-exclude_paths: []
+exclude_paths:
+  - ".github/styles/**"
```

Comment:
@prcr I should do the same for Pulse, right?

Reply:
Yes, that's where I got the idea to ignore the Vale styles since these are files that we typically don't edit directly.

I'm still testing if this works, but if it does, you can apply the same change on https://github.com/codacy/pulse-user-docs/pull/167. I'll let you know! :raised_hands:

Reply:
Actually, I think it's better to test if this works on your pull request @claudiacarpinteiro. If after applying this change the issues reported by Codacy disappear, then it's working as expected.